### PR TITLE
enquote index column name in PostgresFullPrunedBlockStore.java 

### PR DESCRIPTION
**Pull request by mancini0**
enquote index column name in PostgresFullPrunedBlockStore.java to make compatible with Cockroach-DB

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/store/PostgresFullPrunedBlockStore.java [hunk 1]
```diff
@@ -81,7 +81,7 @@
             ")\n";
```
Comment (schildbach) [inline core/src/main/java/org/bitcoinj/store/PostgresFullPrunedBlockStore.java:81]: There is one more occurence in the line above. Does that need to be escaped, too?

**Comment by schildbach** [comment]
My understanding of this change is that it's 100% compatible for Postgres users who use the all-lowercase spelling of `index`. Am I right?

File: core/src/main/java/org/bitcoinj/store/PostgresFullPrunedBlockStore.java (hunk 1 above)
Comment (mancini0) [reply core/src/main/java/org/bitcoinj/store/PostgresFullPrunedBlockStore.java:81]: yes, thanks

**Comment by mancini0** [comment]
> My understanding of this change is that it's 100% compatible for Postgres users who use the all-lowercase spelling of `index`. Am I right?

yes that is correct

**Pull request closed without merging**